roles/openshift_preflight/facts/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: set common variables
+  set_fact:
+    deployment_type: "{{ deployment_type | default('openshift-enterprise') }}"
```

Comment:
@brenton any reason we couldn't bake some of the variables into the container image? We could default deployment_type easily during image build, with the OCP images defaulting to `openshift_enterprise` and the Origin images defaulting to `origin`? We could use this for defaulting version numbers as well (pull the image/tag related to the version you want to install)